hello-world/ACI/app/Dockerfile

```diff
@@ -1,15 +1,23 @@
-FROM alpine:3.19
-RUN apk update && apk add python3 py3-pip --no-cache
-RUN pip3 install flask --no-cache-dir --break-system-packages
+FROM mcr.microsoft.com/devcontainers/base:dev-ubuntu-22.04 AS build
+ENV PROJECT_DIR=~/
+WORKDIR ${PROJECT_DIR}/confidential-container-demos
+COPY . ./
+RUN chmod +x hello-world/get_verbose_report.sh && ./hello-world/get_verbose_report.sh
+
+FROM mcr.microsoft.com/cbl-mariner/base/python:3.9
+RUN pip install flask --no-cache-dir
 
 # all commands start from this directory
 WORKDIR /app
 
 # copy all files from this folder to working directory (ignores files in .dockerignore)
-COPY Dockerfile main.py verbose-report /app/
+ENV BUILD_DIR=~/confidential-container-demos/hello-world/ACI/app
+COPY --from=build ${BUILD_DIR}/Dockerfile ${BUILD_DIR}/main.py ${BUILD_DIR}/verbose-report /app/
+
+RUN chmod +x verbose-report
 
 EXPOSE 80
-ENV PYTHONUNBUFFERED=1
+ENV PYTHONUNBUFFERED=1 FLASK_DEBUG=1
```

Comment:
I think we can leave off `FLASK_DEBUG_1`.  This can be set at runtime if there is an issue to debug.